sdk/communication/communication-phone-numbers/test/public/siprouting/utils/recordedClient.ts

```diff
@@ -117,3 +119,18 @@ export async function createRecordedClientWithToken(
   // casting is a workaround to enable min-max testing
   return { client, recorder };
 }
+
+export async function clearSipConfiguration(): Promise<void> {
+  const client = new SipRoutingClient(env.COMMUNICATION_LIVETEST_DYNAMIC_CONNECTION_STRING ?? "");
```

Comment:
probably should log/throw if the var is empty, since I can't imagine it would do anything useful

Reply:
Currently this is handled by the SDK client. If the variable does not exist, is empty or not valid this will be thrown:
`Error: Invalid connection string`

`env.COMMUNICATION_LIVETEST_DYNAMIC_CONNECTION_STRING ?? ""`
replaced by
`assertEnvironmentVariable("COMMUNICATION_LIVETEST_DYNAMIC_CONNECTION_STRING")`